### PR TITLE
Basic titles and weights fix, and one pro fix

### PR DESCRIPTION
**Pull request by ethangj**

### CODE REVIEW

File: _basic/builds-and-configuration/ignore-command-on-branch.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Include / Exclude Commands On Specific Branches
```
Comment (mlocher) [inline _basic/builds-and-configuration/ignore-command-on-branch.md]: Please capitalize this title as well

File: _basic/builds-and-configuration/root-level-access.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Sudo and Root On Codeship Basic
```
Comment (mlocher) [inline _basic/builds-and-configuration/root-level-access.md]: Please capitalize this title as well

File: _basic/builds-and-configuration/ignore-command-on-branch.md (hunk 1 above)
Comment (ethangj) [inline _basic/builds-and-configuration/ignore-command-on-branch.md]: This one is capitalized?

Comment (mlocher) [inline _basic/builds-and-configuration/ignore-command-on-branch.md]: Either I am blind or I added this comment to the wrong article, can't find the other one though
/shrug

**Comment by mlocher** [comment]
CI is failing because of the merged changes from #730, this is still good to merge

**Pull request merged**